packages/ckeditor5-media-embed/src/mediaembed.js

```diff
@@ -236,6 +236,24 @@ export default class MediaEmbed extends Plugin {
  * @member {Array.<String>} module:media-embed/mediaembed~MediaEmbedConfig#removeProviders
  */
 
+/**
+ * Customizing semantic element name.
```

Comment:
This documentation should link to [**`previewsInData`**](https://ckeditor.com/docs/ckeditor5/latest/api/module_media-embed_mediaembed-MediaEmbedConfig.html#member-previewsInData) and explain the relation between these two (one does not make sense when the other is set `true`).